routines/epochtal.js

```diff
@@ -263,7 +263,7 @@ async function releaseMap (context) {
       fs.rmSync(portal2.output, { recursive: true });
     }
 
-    fs.renameSync(vmf, `${__dirname}/../vmfs/${context.data.week.map.id}.vmf.xz`);
+    fs.renameSync(vmf, context.file.mapvmf);
```

Comment:
I don't think the implementation of VMF archiving should've been changed like this.

In the currently deployed system, VMF files for _all gameplay-relevant maps_ handled by the system are stored in their own directory, named after the workshop item ID, so that we have backup references for them in case that map gets removed from the workshop. It seems like this is assuming that there will only ever be one VMF per context, but doesn't account for contexts with multiple maps, or different contexts which share the same map (thus causing duplicates). This will cause issues when rolling out a tournament that requires a series of maps.

I don't really mind VMF backups being reimplemented in some way, as long as these considerations are kept in mind.